src/sources/fluent/mod.rs

```diff
@@ -110,7 +117,9 @@ impl SourceConfig for FluentConfig {
             cx,
             self.acknowledgements,
             self.connection_limit,
-            None,
+            self.permit_origin
+                .clone()
+                .map(|p| p.0.iter().map(|net| net.0).collect()),
```

Comment:
This is a common enough pattern that we should implement:

```rust
impl From<IpAllowlistConfig> for Vec<IpNet>
```

Then we can:

```suggestion
            self.permit_origin
                .clone()
                .map(Into::into),
```

Reply:
Yes, thank you! I rushed it a bit so I overlooked that :smile: